CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - manipulation ProvideInformationAboutNextCalibration for devices
 - manipulation SetSystemContextActivationStateAndContextAssociation for combined settings
+- manipulation AssociateValidateAndChangeIdentificationOfContextState for contexts
```

Comment:
According to the format in the changelog: The latest version comes first. Please move the line to the top.

Reply:
I am afraid the Guiding Principle "the latest version comes first" of [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) does not say anything about the ordering of entries within a version.

Also: I consider it helpful when the order of entries reflects the order in which the PRs are merged to main. I dont't see why we should break with this property.

Reply:
Adapted the order anyway.